README.md

```diff
@@ -76,37 +75,86 @@ let dog = new Dog("Spot");
+_copied_ are fundamentally a code reuse mechanism--a way for objects to share code. The way that
+the code is shared matters. `Car`looks to the parent as a reference, and does not
+have all `Vehicle`'s properties and behaviors without it. When you attempt to access a property
+or method of an object, JavaScript will continue searching through the object/`class` and its
+`prototype` until the end of the `prototype` chain is reached. It can create performance and
+code maintenance issues when misapplied.
 
-- You probably won't use inheritance often, but as your code bases get larger, you are more
-likely to find a need for it. If you're creating a number of objects that have similar
-features, then creating a generic object type to contain all the shared functionality,
-and inheriting those features in more specialized objects can be convenient and useful.
+```js
+class Vehicle { // Parent class
+  constructor(name) {
+    this.name = name;
+  }
+
+  start() { // Parent class method
+    return "Engine of "+this.name + " starting...";
+  };
+}
+
+class Car extends Vehicle { // Child class
+ constructor(name) {
+    super(name);
+  }
+
+  run() { // Child class method
+    return `Hello! ${super.speak()} It has 4 Cylinders.`; // Calling on a method inherited from parent class
+  }
+}
+
+let car1 = new Car("Camry");
+let car2 = new Car("Civic");
+
+// Accessing the child class method which accesses the parent's method
+car1.run();   // "Hello! Engine of Camry starting... It has 4 Cylinders."
+car2.run();   // "Hello! Engine of Civic starting... It has 4 Cylinders."
+```
 
-- When creating a parent-child `class` relation, it is _semantic_ rather than merely
-_syntactic_. Let's use `Vehicle` as a parent `class` with `Car` as a child
-`class` of `Vehicle` as an example. In _classical_ inheritance, it creates a _copy_ of the
-behavior from parent. However, in JavaScript when we create the object it does not copy the
-properties or behavior, it creates a _link_. `class`es should _never_ be extended unless it
-is for the sake of compatibility with newer JavaScript features. This is due to the way
-inheritance works on top of vanilla JavaScript code.
+It it advised that `class`es should _never_ be extended unless it is for the sake of compatibility
+with newer JavaScript features, like building UI components frameworks in React and Angular.
+This is due to the way inheritance works on top of vanilla JavaScript code, and could create
+a multitude of unforseen headaches down the road. Always start with the simplest solution
+and progress to more complex solutions only as-needed.
 
-- When using inheritance, it is advised to not have too many levels of inheritance, and to keep
-careful track of where you define your methods and properties. Too much inheritance can lead
-to confusion and difficulties trying to debug code.
+If you do choose to use inheritance, it is advised to not have too many levels of inheritance, and
```

Comment:
This could now go into the section behavioral subtyping - maybe because of the subtyping design principle, it is advised you try to stick to as few levels of inheritance as possible. The more levels, the harder it is to maintain the behavioral subtyping